Makefile

```diff
@@ -62,7 +62,7 @@ ifdef NO_DOCKER
 	scBuildImageTarget =
 else
 	# Mount .pkg as pkg so that we save our cached "go build" output files
-	DOCKER_CMD = docker run --rm -ti -v $(PWD):/go/src/$(SC_PKG) \
+	DOCKER_CMD = docker run --rm -ti --net=host -v $(PWD):/go/src/$(SC_PKG) \
```

Comment:
checked and not needed. From an earlier run when I was trying to use our already existing in repo `contrib/hack/start-server.sh`